python-frontend/src/main/java/org/sonar/python/semantic/v2/types/TrivialTypeInferenceVisitor.java

```diff
@@ -96,13 +98,15 @@ public class TrivialTypeInferenceVisitor extends BaseTreeVisitor {
 
   private final TypeTable projectLevelTypeTable;
   private final String fileId;
+  private final String fullyQualifiedModuleName;
```

Comment:
We could store it as a list of strings

Reply:
I actually use the string in my other PR to introduce FQNs for `FunctionType`, so I'd rather keep the string here and split it when needed in the import resolution.